cpg-language-java/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/java/JavaLanguageFrontendTest.kt

```diff
@@ -800,4 +797,65 @@ internal class JavaLanguageFrontendTest : BaseTest() {
         assertNotNull(jArg)
         assertContains(jArg.prevDFG, loopVariable)
     }
+
+    @Test
+    fun testArithmeticOperators() {
+        val file = File("src/test/resources/Issue1444.java")
+
+        val language = JavaLanguage()
+        val result =
+            TestUtils.analyze(listOf(file), file.parentFile.toPath(), true) {
+                it.registerLanguage(language)
+            }
+        val tu =
+            findByUniqueName(result.components.flatMap { it.translationUnits }, file.toString())
+
+        val record = tu.getDeclarationAs(0, RecordDeclaration::class.java)
```

Comment:
instead of using findByUniqueName, getDeclarationAs, etc. you should use our "new" Kotlin-based shortcut API. Unfortunately, almost all unit test in the java frontend still use the old syntax, but you can for example look at the Go frontend for some inspiration.

There is some documentation on how to use it also on the PR that introduced it: https://github.com/Fraunhofer-AISEC/cpg/pull/900

You can use it to quickly retrieve nodes by their name, e.g. `tu.functions["main"]`

Reply:
Right, that's already much nicer!